shell.nix

```diff
@@ -198,14 +104,13 @@
       icu-uc = super.icu;
 
       codexShell = self.haskellPackages.shellFor {
-        buildInputs = [
-          self.haskellPackages.tasty-discover
```

Comment:
Not tasty-discovering any more?

Reply:
Despite my best efforts I could not make `cabal-install` clue in to the nix-provided tasty-discover package, or binary. So I just ripped it out for now.